KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -784,25 +852,53 @@ else if (cards[currentPlayer].get(i) == type) {
     }
 
     /**
-     * Sort a player's cards using an insertion sorting algorithm
-     *
+     * Sort a player's list of cards using a recursive quick sort algorithm
      * @param player The player who's cards will be sorted
+     * @param left The left bounds of the segment to sort (used to recursion, set to 0)
+     * @param right The right bounds of the segment to sort (used to recursion, set to length of array - 1)
+     * @return 
      */
-    public void insertionSort(int player) {
-
-        // Get the player's card list
+    private void quickSortCards(int player, int left, int right) {
```

Comment:
Quik sort is spelt wrong but I don't feel like changing it so it's fine.

Reply:
ive always spelt it with a c
look at the image in the chat

Reply:
nvm I'm just being smooth.